test/fixtures/react.tsx

```diff
@@ -1,7 +1,10 @@
-import { SeamProvider } from '@seamapi/react'
+import 'isomorphic-fetch'
```

Comment:
Weird, I wonder what changed that now requires this import?

Reply:
yea no idea, `fetch` was suddenly not found